contracts/access/ZNSAccessController.sol

```diff
@@ -34,6 +34,8 @@ contract ZNSAccessController is AccessControl, ZNSRoles, IZNSAccessController {
         _setRoleAdmin(GOVERNOR_ROLE, GOVERNOR_ROLE);
         // all of the admins control registrar
         _setRoleAdmin(REGISTRAR_ROLE, ADMIN_ROLE);
+        // all of the admins control domain token
+        _setRoleAdmin(DOMAIN_TOKEN_ROLE, ADMIN_ROLE);
```

Comment:
hmm.... this will require us to upgrade every single contract in ZNS for this change to work? is it possible to do this in a less invasive way?